swagger/swagger.json

```diff
@@ -1035,6 +1217,156 @@
         }
       }
     },
+    "DirectDebitConnectorPaymentResponse" : {
+      "type" : "object",
+      "properties" : {
+        "amount" : {
+          "type" : "integer",
+          "format" : "int64",
+          "readOnly" : true
+        },
+        "state" : {
+          "readOnly" : true,
+          "$ref" : "#/definitions/DirectDebitPaymentState"
+        },
+        "description" : {
+          "type" : "string",
+          "readOnly" : true
+        },
+        "reference" : {
+          "type" : "string",
+          "readOnly" : true
+        },
+        "mandate_id" : {
+          "type" : "string",
+          "readOnly" : true
+        },
+        "payment_id" : {
+          "type" : "string",
+          "readOnly" : true
+        },
+        "payment_provider" : {
+          "type" : "string",
+          "readOnly" : true
+        },
+        "created_date" : {
+          "type" : "string",
+          "readOnly" : true
+        },
+        "provider_id" : {
+          "type" : "string",
+          "readOnly" : true
+        }
+      }
+    },
+    "DirectDebitPayment" : {
+      "type" : "object",
+      "properties" : {
+        "amount" : {
+          "type" : "integer",
+          "format" : "int64",
+          "example" : 1200
```

Comment:
@oswaldquek @stephencdaly I agree we should add more descriptions and more detail (both here and in card payments). But to avoid holding this up, suggest we leave this section as is for now, and look at improving it when we work on the planned next iteration of the direct debit tech docs. 
As a quick fix in the meantime - in the other places that @stephencdaly noted there were missing descriptions, I've copy-pasted descriptions from elsewhere in the Swagger file.